Periods control

butofleury opened the pull request.

mestachs commented:
Can you explain what is this doing ?

mestachs commented on package.json:
```diff
@@ -53,20 +53,24 @@
-  ]
+  ],
+  "dependencies": {
+    "formik": "^1.5.1",
```
where is this used ?

butofleury replied:
sorry, I had not noticed that I added it in commit. I will delete it.

butofleury commented:
> Can you explain what is this doing ?

1. On burundi project, some invoices are generated only for the first month or the 2 last months of the quarter. So, we added a control on that to allow that kind of payment. 

2. We also changed the invoice buttons links component to allow viewing the invoices in the box Dialog. Because some org units are linked to 7 or 8 invoices. 

3. We added also the managment of category combo as org units, to allow display the donors in the org unit list.

mestachs commented:
I'm not a big fan of the button to select the factures.

it makes the invoices less discoverable/greppable for us (switching between projects is easer) and for clients (at least for first usage we don't hide info behind a button)

can't we trigger a behaviour where we show the previous format when not "too much invoices"
or show the 3 first invoices like before and a "more" button

IMO the label should be "Choisir une facture"
